packages/core/src/lib/PlanCard/types.ts

```diff
@@ -33,4 +34,25 @@ export interface IPlanCardProps {
      * the description title component of the Card
      */
     descriptionTitle: string;
+
+    /**
+     * a horizontal line on the top
+     */
+    horizontalLine: boolean;
+
+    /**
+     * border color
+     */
+    borderColor: typeof color | string;
+
+    /**
+     * Card background color
+     */
+    backgroundColor?: typeof color | string;
```

Comment:
for `typeof color` to work inside the component you have to use `color[backgroundColor]` since that is not being done this can be removed since we are are also taking string